src/View/PdfView.php

```diff
@@ -12,18 +12,18 @@
  */
 namespace CakePdf\View;
 
+use App\View\AppView;
 use CakePdf\Pdf\CakePdf;
 use Cake\Core\Configure;
 use Cake\Core\Exception\Exception;
 use Cake\Event\EventManager;
 use Cake\Network\Request;
 use Cake\Network\Response;
-use Cake\View\View;
 
 /**
- * @package       Cake.View
+ * Renders PDFs instead of HTML.
  */
-class PdfView extends View
+class PdfView extends AppView
```

Comment:
Yes, we will have to wait for 3.3: https://github.com/cakephp/cakephp/issues/8155
But we can already require it for this plugin in a new version, as long as we adjust the documentation as well.